src/test/java/bio/terra/app/controller/DatasetValidationsTest.java

```diff
@@ -44,7 +44,7 @@
 import static org.hamcrest.CoreMatchers.equalTo;
 import static org.hamcrest.Matchers.containsInAnyOrder;
 import static org.junit.Assert.assertArrayEquals;
-import static org.junit.Assert.assertThat;
+import static org.hamcrest.MatcherAssert.assertThat;
```

Comment:
Oh wow! Thanks for going this!  I'm surprised that it isn't complaining about import order...